app/controllers/project_instances_controller.rb

```diff
@@ -59,9 +59,12 @@ def update
   end
 
   def new
-    project_id = Project.where("url like ?", "%#{params[:nagios_url]}%").first.try(:id) if params[:nagios_url]
+    nagios_url = params[:nagios_url].presence || ""
```

Comment:
not sure if we need the `|| ""` here. If nagios_url is nil, we don't really want to run the search in line 63, but with this in place the query will run and potentially match on any project - why?